snippets/markdown/audio/tensorflowjs/javascript.md

```diff
@@ -1,53 +1,58 @@
 ```html
-<script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs@1.1.2/dist/tf.min.js"></script>
-<script src="https://cdn.jsdelivr.net/npm/@tensorflow-models/speech-commands@0.3.8/dist/speech-commands.min.js"></script>
+<div>Teachable Machine Audio Model</div>
+<button type='button' onclick='init()'>Start</button>
+<div id='label-container'></div>
+<script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs@1.3.1/dist/tf.min.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/@tensorflow-models/speech-commands@0.4.0/dist/speech-commands.min.js"></script>
 
 <script type="text/javascript">
     // more documentation available at
     // https://github.com/tensorflow/tfjs-models/tree/master/speech-commands
 
-    // the json file (model topology) has a reference to the bin file (model weights)
-    const checkpointURL = '{{MODEL_URL}}';
-    // the metatadata json file contains the text labels of your model and additional information
-    const metadataURL = '{{METADATA_URL}}';
+    // the link to your model provided by Teachable Machine export panel
+    const modelURL = '{{MODEL_URL}}';
 
-    const recognizer = speechCommands.create(
-        'BROWSER_FFT',
-        undefined,
-        checkpointURL,
-        metadataURL);
+    async function createModel() {
+        const delimiter = (modelURL.charAt(modelURL.length - 1) !== '/') ? '/' : '';
```

Comment:
`{{URL}}` its always with a trailing `/`, so you don't need to have this check

Reply:
Yep you're right. I was copying from the image one, where it's in the internal load function. But for audio it's pretty obvious since there's no load function abstracting the functionality.